lib/ssl/src/tls_handshake_1_3.erl

```diff
@@ -552,13 +560,14 @@ build_content(Context, THash) ->
 do_start(#client_hello{cipher_suites = ClientCiphers,
                        session_id = SessionId,
                        extensions = Extensions} = _Hello,
-         #state{connection_states = _ConnectionStates0,
+         #state{connection_states = ConnectionStates0,
                 ssl_options = #{ciphers := ServerCiphers,
                                 signature_algs := ServerSignAlgs,
                                 supported_groups := ServerGroups0,
                                 alpn_preferred_protocols := ALPNPreferredProtocols,
                                 honor_cipher_order := HonorCipherOrder},
-                session = #session{own_certificate = Cert}} = State0) ->
+                session = #session{own_certificate = Cert}} = State00) ->
```

Comment:
It would be easier to read the code if the state variables would follow the same naming convention: starting with State0 and incrementing all other occurrences.